dali/util/nvml.h

```diff
@@ -310,6 +310,25 @@ inline bool HasCuda11NvmlFunctions() {
   return nvmlHasCuda11NvmlFunctions();
 }
 
+inline bool isHWDecoderSupported() {
+  if (nvml::HasCuda11NvmlFunctions()) {
+    unsigned int device_count;
+    CUDA_CALL(nvmlDeviceGetCount_v2(&device_count));
+    for (unsigned int device_idx = 0; device_idx < device_count; device_idx++) {
+      auto info = nvml::GetDeviceInfo(device_idx);
+      std::cerr << "Device " << device_idx
+                << " brand " << info.type
+                << " cc_M " << info.cap_major
+                << " cc_m " << info.cap_minor
+                << std::endl;
```

Comment:
Out of curiosity, why are we printing the results?

Reply:
It was a part of the test. Moved back there.